### DISCUSSION TITLE
Install error (jax[cuda]) on Ubuntu 20.04.4

### DISCUSSION THREAD

**Discussion by fa3d** [jax-ml/jax#11269] [Q&A] [answered]
I am tying to install using the commend:

`pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html`

but I get the error:

> Looking in links: https://storage.googleapis.com/jax-releases/jax_releases.html
> Collecting jax[cuda]
>   Using cached jax-0.3.13.tar.gz (951 kB)
> Requirement already satisfied, skipping upgrade: absl-py in /home/f/.local/lib/python3.8/site-packages (from jax[cuda]) (1.1.0)
> Requirement already satisfied, skipping upgrade: numpy>=1.19 in /home/f/.local/lib/python3.8/site-packages (from jax[cuda]) (1.23.0)
> Collecting opt_einsum
>   Using cached opt_einsum-3.3.0-py3-none-any.whl (65 kB)
> Requirement already satisfied, skipping upgrade: scipy>=1.2.1 in /home/f/.local/lib/python3.8/site-packages (from jax[cuda]) (1.8.1)
> Requirement already satisfied, skipping upgrade: typing_extensions in /home/f/.local/lib/python3.8/site-packages (from jax[cuda]) (4.2.0)
> **ERROR: Could not find a version that satisfies the requirement jaxlib==0.3.10+cuda11.cudnn82 (from jax[cuda]) (from versions: 0.1.32, 0.1.40, 0.1.41, 0.1.42, 0.1.43, 0.1.44, 0.1.46, 0.1.50, 0.1.51, 0.1.52, 0.1.55, 0.1.56, 0.1.57, 0.1.58, 0.1.59, 0.1.60, 0.1.61, 0.1.62, 0.1.63, 0.1.64, 0.1.65, 0.1.66, 0.1.67, 0.1.68, 0.1.69, 0.1.70, 0.1.71, 0.1.72, 0.1.73, 0.1.74, 0.1.75, 0.1.76, 0.3.0, 0.3.2, 0.3.5, 0.3.7, 0.3.8, 0.3.10)
> ERROR: No matching distribution found for jaxlib==0.3.10+cuda11.cudnn82 (from jax[cuda])**

I have Ubuntu 20.04.4 with NVIDIA Driver Version: 515.48.07 and  CUDA Version: 11.7.  
I have Python 3.8.10

Please help. I have no clue what to do

**Comment by YouJiacheng** [comment, marked as answer]
CUDA wheels are now at jax_cuda_releases.html rather than jax_releases.html.
Use `pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html`
